recipes/tk/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set version = "8.5.19" %}
+{% set maj_min = ".".join(version.split(".")[:2]) %}
+
+package:
+  name: tk
+  version: {{ version }}
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+
+requirements:
+  build:
+    - python         # [win]
+    - curl           # [win]
+    - 7za            # [win]
+
+test:
+  requires:
+    - python 2.7.*   # [win and py27]
+    - python 3.4.*   # [win and py34]
+    - python 3.5.*   # [win and py35]
+
+  files:
+    - hello.tcl
+
+  commands:
+    # Check for binaries.
+    {% for ver_suffix in ["", maj_min] %}
+    - test -f "${PREFIX}/bin/tclsh{{ ver_suffix }}"                           # [unix]
+    - test -f "${PREFIX}/bin/wish{{ ver_suffix }}"                            # [unix]
+    - if not exist %LIBRARY_PREFIX%\\bin\\tclsh{{ ver_suffix }}.exe exit 1    # [win]
```

Comment:
Yeah, looks like they are not installing (and possibly building) `tclsh` on Windows for some reason. Not sure why. They do install `wish` though, which we can use.